compute_tools/src/compute.rs

```diff
@@ -133,6 +133,60 @@ impl TryFrom<ComputeSpec> for ParsedSpec {
     }
 }
 
+/// Create special neon_superuser role, that's a slightly nerfed version of a real superuser
+/// that we give to customers
+fn create_neon_superuser(spec: &ComputeSpec, client: &mut Client) -> Result<()> {
+    let roles = spec
+        .cluster
+        .roles
```

Comment:
When new role / db is created, it's present in this list, but not in Postgres, yet. So grant bellow will fail because `create_neon_superuser` is executed before `handle_roles` and co

That's why e2e failed pretty consistently in all cases, I guess

Reply:
Ah that's a good catch. I'll update the query to only update those roles that already exist